runrestic/runrestic/configuration.py

```diff
@@ -122,7 +121,15 @@ def configuration_file_paths() -> Sequence[str]:
 def parse_configuration(config_filename: str) -> Dict[str, Any]:
     logger.debug(f"Parsing configuration file: {config_filename}")
     with open(config_filename) as file:
-        config = toml.load(file)
+        if config_filename.endswith('.json'):
+            import json
+            config = toml.load(file)
+        if config_filename.endswith(('yaml','.yml')):
+            import yaml
+            config = toml.load(file)
```

Comment:
that doesn't seem right. `toml.`

do you acutely need this? I mean it's nice and all but it does add extra maintenance for a problem I hadn't had this far.

Reply:
I updated the commit.